src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -92,6 +93,7 @@ private void CloseButton_Click(object sender, RoutedEventArgs e)
                 Actions.Close,
                 Categories.Preferences);
             viewModel.PackagePathsViewModel.SaveSettingCommand.Execute(null);
+            viewModel.CommitPackagePathsForInstall();
```

Comment:
it's a bit weird to me that some of this logic lives in `PackagePathsViewModel` and some on `PreferencesViewModel` directly, guess it would be nice if it all lived in PackagePathViewModel and was one method we could call.

Reply:
I leave that for a future refactoring.